manifests/istio-telemetry/kiali/templates/service.yaml

```diff
@@ -3,10 +3,15 @@ kind: Service
 metadata:
   name: kiali
   namespace: {{ .Release.Namespace }}
+  annotations:
```

Comment:
per john's comment this strikes me as the wrong approach - we should not be adding things that are in IOP to values.yaml.

Reply:
But I think for now we should keep it consistent with other templates, which means that we should have it before we fully remove helm support.

Reply:
we are quite likely to move the kiali chart out of the repo shortly so this is not likely to be a lasting change. does this pr work without this change? if not we will have to tackle this problem sooner or later.

Reply:
without this, user can still do overlay via IOP API, they just cannot use the helm values pass through, but we actually can do that for other addons. I think we'd better to remove all of them at once to keep it consistent later